testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -3946,6 +3946,87 @@ static void checkRlikeFails(SqlOperatorFixture f) {
     f.checkNull("ASCII(cast(null as varchar(1)))");
   }
 
+  @Test void testParseUrl() {
```

Comment:
I will put this method in parseTime after, making it more reasonable. But it is Ok to put it here.

Reply:
Thanks for reviewing. I have considerd this option, but these tests are related to date/time/timestmap processing, not string processing, so I place it here.